src/spikeinterface/preprocessing/tests/test_decimate.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+from pathlib import Path
+
+import itertools
+from spikeinterface import NumpyRecording
+from spikeinterface.core import generate_recording
+from spikeinterface.preprocessing.decimate import DecimateRecording
+import numpy as np
+
+
+@pytest.mark.parametrize("decimation_offset", [0, 1, 9, 10, 11, 100, 101])
```

Comment:
Thanks! On the same line, shouldn't `decimation_offset=101` fail?

In #1989 we (kinda) agreed that we shouldn't allow for empty arrays returns. So if you have 101 samples and you ask for `parent_traces[101::*]` it will return an empty array.

Reply:
Not sure what you mean... I check that some parameter combinations raise ValueErrors [here](https://github.com/SpikeInterface/spikeinterface/blob/6b07f9423d7954fb3d49c6f4b4b859c2e3c031de/src/spikeinterface/preprocessing/tests/test_decimate.py#L22) (couldn't find an elegant way to do it with `parameterize`

Reply:
totally missed that! Looks great!